packages/semver/src/executors/version/version.ts

```diff
@@ -83,16 +80,17 @@ export function versionWorkspace({
           )
         )
       ),
-      concatMap((packageFiles) =>
-        addToStage({
-          paths: packageFiles.filter(
-            (packageFile) => packageFile !== null
-          ) as string[],
-          dryRun,
-        })
-      )
+      concatMap((packageFiles) => of({
+        paths: packageFiles.filter(
+          (packageFile) => packageFile !== null
+        ) as string[],
+        dryRun,
+      }))
     ),
   ]).pipe(
+    concatMap((changesToStage) =>
+      addToStage(mergeAddToStage(changesToStage, dryRun))
+    ),
```

Comment:
So here I would keep the old signature:

```suggestion
    concatMap((paths) =>
      addToStage({ path, dryRun })
    ),
```

Reply:
Yea, this is what I’m thinking is better, to just have the previous stage pass in the paths only, since dry run isn’t necessary in those returned objects